content/en/docs/installation/armory-deployments-for-spinnaker.md

```diff
@@ -159,6 +163,15 @@ kubectl create ns armory-agent
 
 > The examples on this page assume you are using a namespace called armory-agent for the Agent. Replace the namespace in the examples if you are using a different namespace.
 
+#### Install Argo Rollouts
```

Comment:
This is now in the "manual" section for those that don't want to use Helm.

Reply:
Seems odd to be me that someone would say "nope don't want to use helm for this" when we're already telling them to use Helm to install the Agent. I'd be in favor of just ripping it out and saying the Helm chart installs everything you need, including Cloud Agent, Argo Rollouts, and (whatever ele we mght add in the future).

Will review the whole thing later today or Monday.

Reply:
Talked to Stephen about this. There are instances where a customer might already have ArgoRollouts

Reply:
To close this loop, the mega helm chart will be the recommended path. The manual Agent install instructions are being moved to the KB.